packages/networkx/meta.yaml

```diff
@@ -0,0 +1,16 @@
+package:
+  name: networkx
+  version: '2.2'
+
+source:
+  url: https://files.pythonhosted.org/packages/f3/f4/7e20ef40b118478191cec0b58c3192f822cace858c19505c7670961b76b2/networkx-2.2.zip
+  sha256: 45e56f7ab6fe81652fb4bc9f44faddb0e9025f469f602df14e3b2551c2ea5c8b
+
+requirements:
+  run:
+    - decorator
+    - setuptools
+
+test:
+  imports:
+    - networkx
```

Comment:
Could you copy-past other imports from  https://github.com/conda-forge/networkx-feedstock/blob/master/recipe/meta.yaml ?

Reply:
I pasted in all the imports except `.tests` and `.testing`